hazelcast/src/main/java/com/hazelcast/jet/impl/AbstractJetInstance.java

```diff
@@ -146,7 +146,7 @@ public LightJob newLightJob(DAG dag) {
     }
 
     @Nonnull
-    protected abstract LightJob newLightJobInt(Object jobDefinition);
+    public abstract LightJob newLightJobInt(Object jobDefinition);
```

Comment:
@viliam-durina are you ok with this particular change?

Reply:
In non-public AbstractJetInstance it's ok i guess